1.8.3/arm64/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# openhab image
+#
+# ------------------------------------------------------------------------------
+#               NOTE: THIS DOCKERFILE IS GENERATED VIA "update.sh"
+#
+#                       PLEASE DO NOT EDIT IT DIRECTLY.
+# ------------------------------------------------------------------------------
+#
+
+FROM multiarch/debian-debootstrap:arm64-jessie
+
+# Set download urls
+ENV OPENHAB_URL="https://bintray.com/artifact/download/openhab/bin/distribution-1.8.3-runtime.zip"
+ENV JAVA_URL="https://www.azul.com/downloads/zulu/zdk-8-ga-linux_aarch32hf.tar.gz"
+
+# Set variables
+ENV \
+    APPDIR="/openhab" \
+    EXTRA_JAVA_OPTS="" \
+    JAVA_HOME='/usr/lib/java-8' \
+    OPENHAB_HTTP_PORT="8080" \
+    OPENHAB_HTTPS_PORT="8443" \
+    USER_ID="9001"
+
+# Basic build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG VCS_REF
+LABEL org.label-schema.build-date=$BUILD_DATE \
+    org.label-schema.docker.dockerfile="/Dockerfile" \
+    org.label-schema.license="EPL" \
+    org.label-schema.name="openHAB" \
+    org.label-schema.url="http://www.openhab.com/" \
+    org.label-schema.vcs-ref=$VCS_REF \
+    org.label-schema.vcs-type="Git" \
+    org.label-schema.vcs-url="https://github.com/openhab/openhab-docker.git"
+
+# Install basepackages
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+      ca-certificates \
+      unzip \
+      wget \
+      && rm -rf /var/lib/apt/lists/*
+
```

Comment:
OK, I found it.
The update.sh script is somewhat more complicated, you have to adjust, since the volume folders are different and the openhab installation. Thank you!